source/ada/lsp-ada_handlers.adb

```diff
@@ -2874,45 +2890,80 @@ package body LSP.Ada_Handlers is
             end loop;
          end Process_Comments;
 
-      begin
-         if Name_Node = No_Name then
-            return;
-         end if;
+         use Laltools.Refactor.Rename;
 
-         Definition := Laltools.Common.Resolve_Name
-           (Name_Node,
-            Self.Trace,
-            Imprecise => Imprecise);
+         Cursor_1 : Unit_Slocs_Maps.Cursor;
```

Comment:
(style): name the cursors with something more meaningful, for instance `Unit_Cursor` and `Sloc_Cursor` ?